### ISSUE TITLE
Manifest is trying to analyze vim's swap files

### ISSUE THREAD

**Issue by erickt** [nickjj/manifest-revision-webpack-plugin#6]
I use vim, which produces files like `/.foo.js.swp`, but this causes webpack to emit errors like:

```
ERROR in ./client/.entry.js.swp
15:21:45 dev.1      | Module parse failed: /tmp/client/.entry.js.swp Line 1: Unexpected number
15:21:45 dev.1      | You may need an appropriate loader to handle this file type.
15:21:45 dev.1      | (Source code omitted for this binary file)
```

From what I understand, this is coming from the [walkAndPrefetchAssets](https://github.com/nickjj/manifest-revision-webpack-plugin/blob/master/index.js#L115), where it doesn't use the common pattern of optionally including or excluding files based off of a regex. That would let me filter out the `.swp` files, or just explicitly consider the files I would want included. Could this be supported?

**Comment by nickjj** [comment]
Sounds like it would be beneficial to add new options for both including and excluding specific files that supports regex matching. Right now you can optionally ignore paths but not files.

Would you be willing to open a PR on this?

**Comment by jpassaro** [comment]
Any progress on this? I'd be down to tackle as a PR if nobody else has yet.

**Comment by nickjj** [comment]
@jpassaro A patch was merged in a few months ago that added the ability to filter asset types through a regex in PR #19.

It's in the docs https://github.com/nickjj/manifest-revision-webpack-plugin#api and available since `v0.3`.
